src/Arcus.Security.Tests.Integration/HashiCorp/Hosting/HashiCorpVaultTestServer.cs

```diff
@@ -186,8 +201,14 @@ private async Task StartHashiCorpVaultAsync()
         /// </exception>
         public async Task MountKeyValueAsync(string path, VaultKeyValueSecretEngineVersion version)
         {
-            Guard.NotNullOrWhitespace(path, nameof(path), "Requires a path to mount the KeyValue secret engine to");
-            Guard.For<ArgumentException>(() => !Enum.IsDefined(typeof(VaultKeyValueSecretEngineVersion), version), "Requires a KeyValue secret engine version that is either V1 or V2");
+            if (string.IsNullOrWhiteSpace(path))
+            {
+                throw new ArgumentNullException(nameof(path), "Requires a path to mount the KeyValue secret engine to");
+            }
+            if (!Enum.IsDefined(typeof(VaultKeyValueSecretEngineVersion), version))
```

Comment:
```suggestion

            if (!Enum.IsDefined(typeof(VaultKeyValueSecretEngineVersion), version))
```
Best to have a blank line between curly brackets, following the Microsoft code style.